storm-client/src/storm.thrift

```diff
@@ -777,9 +777,14 @@ service Nimbus {
   string getNimbusConf() throws (1: AuthorizationException aze);
   // stats functions
   ClusterSummary getClusterInfo() throws (1: AuthorizationException aze);
+  list<TopologySummary> getTopologySummaries() throws (1: AuthorizationException aze);
+  TopologySummary getTopologySummaryByName(1: string name) throws (1: AuthorizationException aze);
+  TopologySummary getTopologySummaryById(1: string id) throws (1: AuthorizationException aze);
```

Comment:
I think this is better to be `getTopologySummary` (without `ById`) since other functions don't have `ById` when it is getting information by id, e.g. `getTopologyInfo, getTopologyPageInfo, getTopologyConf, getTopology`, etc